InitSSL.java

```diff
@@ -0,0 +1,30 @@
+import static my.HardCodes.*;
+
+import java.io.*;
+import java.security.KeyStore;
+import javax.net.ssl.KeyManagerFactory;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManagerFactory;
+
+public class InitSSL {
+  SSLContext initSsl(String keyLocation) {
+    try {
+      KeyStore keyStore = KeyStore.getInstance(KEY_STORE);
+      keyStore.load(new FileInputStream(keyLocation), PASSWORD.toCharArray());
```

Comment:
What is `PASSWORD` in this case?

Reply:
PASSWORD (and KEY_STORE) are hardcoded values in a class called HardCodes that I have not checked in. I also updated the PR description with more information.